releases/nightly-build/compose-files/docker-compose-nexus-mongo.yml

```diff
@@ -435,21 +435,29 @@ services:
       Service_Host: edgex-app-service-configurable-rules
       MessageBus_SubscribeHost_Host: edgex-core-data
       edgex_registry: consul://edgex-core-consul:8500
+      Binding_PublishTopic: events
     depends_on:
       - consul
       - logging
       - data
 
   rulesengine:
-    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:master
+    image: emqx/kuiper:0.3.0
     ports:
-      - "48075:48075"
-    container_name: edgex-support-rulesengine
-    hostname: edgex-support-rulesengine
+      - "9081:9081"
```

Comment:
Anyway to configure Kuiper to use our standard `48075` rules engine port? Our proxy setup initializes routes to rules engine via this port.